test/aries-js-worker/test/verifiable/verifiable.js

```diff
@@ -121,4 +121,21 @@ async function verifiableStore(newAries, mode = wasmMode) {
             err => done(err)
         )
     })
+
+    it(modePrefix + "Alice generates the verifiable presentation to pass it to the employer", function (done) {
+        aries.verifiable.generatePresentation({
+            "vc": vc
```

Comment:
So the flow you are expecting is for someone to fetch the VC bytes by VCID and then pass those VC bytes back into the generate presentation request?

Reply:
I think that normal flow would be that someone passes VCID, then we retrieve it and create presentation.

Reply:
Yes - didn't want to tie up this API with store. User can recieve the VC from an issuer and use this API directly without storing.

Reply:
Wouldn't the private keys still need to be in the store though?

Reply:
Are you talking about https://github.com/hyperledger/aries-framework-go/issues/1316 ?

Reply:
Actually we need it from friendly name.

Reply:
I'd say it's from ID and we need to be able to query for stored VCs.

Reply:
[Added](https://github.com/hyperledger/aries-framework-go/compare/5a98fe895c8a43e3ce5e007fa7b43eaa33affbc8..a6c0d8c22996e3f81a28be2a908a2e409cbbc94e#diff-307e2ee4ae6edb8abe0a65e7a61b69e6R121) new API to generate presentation from the ID.